packages/pds/src/db/migrations/20230613T164932261Z-init.ts

```diff
@@ -0,0 +1,375 @@
+      'reportId',
+      'actionId',
+    ])
+    .execute()
+  await db.schema
+    .createIndex('moderation_report_resolution_action_id_idx')
+    .on('moderation_report_resolution')
+    .column('actionId')
+    .execute()
+
+  await db.schema
+    .createTable('record')
+    .addColumn('uri', 'varchar', (col) => col.primaryKey())
+    .addColumn('cid', 'varchar', (col) => col.notNull())
+    .addColumn('did', 'varchar', (col) => col.notNull())
+    .addColumn('collection', 'varchar', (col) => col.notNull())
+    .addColumn('rkey', 'varchar', (col) => col.notNull())
+    .addColumn('indexedAt', 'varchar', (col) => col.notNull())
+    .addColumn('takedownId', 'varchar')
+    .execute()
+  await db.schema
+    .createIndex('record_did_cid_idx')
+    .on('record')
+    .columns(['did', 'cid'])
+    .execute()
+  await db.schema
+    .createIndex('record_did_collection_idx')
+    .on('record')
+    .columns(['did', 'collection'])
+    .execute()
+
+  await db.schema
+    .createTable('refresh_token')
+    .addColumn('id', 'varchar', (col) => col.primaryKey())
+    .addColumn('did', 'varchar', (col) => col.notNull())
+    .addColumn('expiresAt', 'varchar', (col) => col.notNull())
+    .addColumn('nextId', 'varchar')
+    .addColumn('appPasswordName', 'varchar')
+    .execute()
+  await db.schema // Aids in refresh token cleanup
+    .createIndex('refresh_token_did_idx')
+    .on('refresh_token')
+    .column('did')
+    .execute()
+
+  await db.schema
+    .createTable('repo_blob')
+    .addColumn('cid', 'varchar', (col) => col.notNull())
+    .addColumn('recordUri', 'varchar', (col) => col.notNull())
+    .addColumn('commit', 'varchar', (col) => col.notNull())
+    .addColumn('did', 'varchar', (col) => col.notNull())
+    .addColumn('takedownId', 'varchar')
+    .addPrimaryKeyConstraint(`repo_blob_pkey`, ['cid', 'recordUri'])
+    .execute()
+  await db.schema // supports rebase
+    .createIndex('repo_blob_did_idx')
+    .on('repo_blob')
+    .column('did')
+    .execute()
+
+  await db.schema
+    .createTable('repo_commit_block')
+    .addColumn('creator', 'varchar', (col) => col.notNull())
+    .addColumn('commit', 'varchar', (col) => col.notNull())
+    .addColumn('block', 'varchar', (col) => col.notNull())
+    .addPrimaryKeyConstraint('repo_commit_block_pkey', [
+      'creator',
+      'commit',
+      'block',
+    ])
+    .execute()
+
+  await db.schema
+    .createTable('repo_commit_history')
+    .addColumn('creator', 'varchar', (col) => col.notNull())
+    .addColumn('commit', 'varchar', (col) => col.notNull())
+    .addColumn('prev', 'varchar')
+    .addPrimaryKeyConstraint('repo_commit_history_pkey', ['creator', 'commit'])
+    .execute()
+
+  await db.schema
+    .createTable('repo_root')
+    .addColumn('did', 'varchar', (col) => col.primaryKey())
+    .addColumn('root', 'varchar', (col) => col.notNull())
+    .addColumn('indexedAt', 'varchar', (col) => col.notNull())
+    .addColumn('takedownId', 'varchar')
```

Comment:
I think we may need to run a manual migration for the live service to change these cols to varchars.

Reply:
Oh good snag 🤔 

Would we rather these be varchars or integers? Does it matter?

Reply:
I can't quite remember why we changed it to a varchar back in the day

Reply:
I think it was so that you can store any identifier there, i.e. if the takedown came from some external system that didn't use integers to identify their actions.  Pretty sure it was foreseeing removing some of the moderation machinery from the PDS, making that field less coupled to the current implementation.

Overall it doesn't matter a ton either way, though 👍